assembl/static2/tests/unit/containers/authentication.spec.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactTestRenderer from 'react-test-renderer';
+import Authentication from '../../../js/app/containers/authentication';
+
+describe('This test concern Authentication component', () => {
+  it('Should test React renderer', () => {
+    const renderer = ReactTestRenderer.create(
+      <Authentication />
+      );
+    const result = renderer.toJSON();
```

Comment:
Hmm, ooh, so the `ReactTestRenderer` renders the entire virtual DOM and flushes the state onto it? So it's not actually binding the elements in the react DOM to the real DOM? So, what runs these tests? A node.js test runner? If so, then these are unit tests only, testing just the logic, and not the browser support and compatibility, right?